### PR TITLE
Service check kb

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Moving this KB to the main doc: https://help.datadoghq.com/hc/en-us/articles/115002137506-Where-can-I-visualize-my-service-check-in-the-Datadog-UI-

### Motivation
Support request

### Preview link
* https://docs-staging.datadoghq.com/gus/service-check-KB/monitors/faq/visualize-your-service-check-in-the-datadog-ui/

### CODE REVIEW

**Comment by l0k0ms** [comment]
@ruthnaebeck would it be possible to remove the KB from the help center and set up a redirection for this page?

**Comment by ruthnaebeck** [comment]
@l0k0ms - Yep. I'll set it up once this is merged. Looks good to me 👍

File: content/monitors/faq/visualize-your-service-check-in-the-datadog-ui.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Configure your custom check monitor: 
+​
+{{< img src="monitors/faq/check_monitor_config.png" alt="Check monitor configuration" responsive="true" >}}
+
+In this form, you can:
+
+* **Pick a custom check**: Select your check status name to monitor
+* **Pick monitor scope**: Select the context for your monitor (including/excluding tags)
+* **Set alert conditions**: Choose between a simple check alert or a cluster alert
+* **Say what's happening**: Edit the notification sent (find more here)
```
Comment (cswatt) [inline content/monitors/faq/visualize-your-service-check-in-the-datadog-ui.md]: Is "find more here" supposed to link to something?

Comment (ruthnaebeck) [reply content/monitors/faq/visualize-your-service-check-in-the-datadog-ui.md]: According to the KB, it should link to:
https://docs.datadoghq.com/monitors/notifications/

**Comment by ruthnaebeck** [comment]
Redirect is now up for the KB article:
https://help.datadoghq.com/hc/en-us/articles/115002137506-Where-can-I-visualize-my-service-check-in-the-Datadog-UI-

**Pull request merged**